<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bot Detective</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Images/Icon/Icons/WindowIcon.png</normaloff>:/Images/Icon/Icons/WindowIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(72, 60, 72);
font: 12pt &quot;Arial&quot;;
color: white;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="MainLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QComboBox" name="SearchBy">
        <property name="styleSheet">
         <string notr="true">color: black;
background-color: rgb(199, 199, 199);
font: 14pt &quot;Arial&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="BotOut">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;
background-color: rgb(199, 199, 199);
font: 14pt &quot;Arial&quot;;</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="Scroll">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>1120</width>
           <height>477</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item>
       <widget class="QLineEdit" name="MessageIn">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;
background-color: rgb(199, 199, 199);
font: 14pt &quot;Arial&quot;;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Send">
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>75</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: black;
background-color: rgb(199, 199, 199);
font: 14pt &quot;Arial&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actHelp"/>
    <addaction name="actDevelops"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="title">
     <string>Language</string>
    </property>
    <addaction name="actionEng"/>
    <addaction name="actionUkr"/>
   </widget>
   <widget class="QMenu" name="menuShow_All">
    <property name="title">
     <string>Show all</string>
    </property>
    <addaction name="actionDentists"/>
    <addaction name="actionSpecializations"/>
    <addaction name="actionDentistry"/>
   </widget>
   <addaction name="menuHelp"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuShow_All"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actDevelops">
   <property name="text">
    <string>Develop</string>
   </property>
  </action>
  <action name="actionEng">
   <property name="text">
    <string>English</string>
   </property>
  </action>
  <action name="actionUkr">
   <property name="text">
    <string>Українська</string>
   </property>
  </action>
  <action name="actionDentists">
   <property name="text">
    <string>Dentists</string>
   </property>
  </action>
  <action name="actionSpecializations">
   <property name="text">
    <string>Specializations</string>
   </property>
  </action>
  <action name="actionDentistry">
   <property name="text">
    <string>Dentistry</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
